pynet/datasets/hcp.py

```diff
@@ -0,0 +1,104 @@
+import os
+from collections import namedtuple
+
+import boto3
+import nibabel as nib
+import numpy as np
+import pandas as pd
+
+Item = namedtuple("Item", ["input_path", "output_path", "metadata_path"])
+
+
+def nii_to_np_arr(filename, low=False, binary=False):
+    img = nib.load(filename)
+    img_data = img.get_data()
+    img_data = np.append(img_data[2:-2, :, 2:-2], np.zeros((256, 9, 256)), axis=1)
+    if low:
+        img_data = img_data.reshape(32, 8, 40, 8, 32, 8).mean(axis=(1, 3, 5))
+
+    if binary:
+        img_data = 1 * (img_data > img_data.mean())
+    return img_data
+
+
+def extract_hcp_data(datasetdir, data, masks, s3, bucket, subject, label, low, metadata):
```

Comment:
General remark we have to make this function as generic as possible. In the future we may use it to retrieve other data.
Simplify the use fct(outdir, s3, bucket, subject, modality, low) -> arr.
Add a docstring to describe the input parameters.
Function name get_hcp_data?
Label is not clear. I think we can replace this by modality.